web/src/beta/ui/layout/Area/index.tsx

```diff
@@ -235,22 +235,19 @@ export const Area = forwardRef<AreaRef, AreaProps>(
         localStorage.setItem(collapsedStorageKey, "0");
       }
       setSize(prevSize => {
-        const width = direction === "column" ? DEFAULT_WIDTH : prevSize.width;
-        const height = direction === "row" ? DEFAULT_HEIGHT : prevSize.height;
+        const w = direction === "column" ? width ?? DEFAULT_WIDTH : prevSize.width;
+        const h = direction === "row" ? height ?? DEFAULT_HEIGHT : prevSize.height;
```

Comment:
Maybe we can use something more descriptive like `previousWidth` or `prevWidth` and `previousHeight` or `prevHeight` in this case?

Reply:
This `width` or `height` here is from component props, which means component user gives an initialWidth or initialHeight, so using `initialWidth` `initialHeight` might be better. I'll update this. Thank you!

Reply:
Got it.